srcpkgs/ofono/template

```diff
@@ -1,22 +1,18 @@
 # Template file for 'ofono'
 pkgname=ofono
-version=1.26
+version=1.28
 revision=1
 build_style=gnu-configure
-configure_args="--disable-bluez4 --enable-external-ell"
```

Comment:
why ?, we have ell available as a system component

Reply:
There were errors during the build that I could only circunvent by removing this flag (which was added to circunvent build errors in other version — 1.26 I guess).

I didn't give enough time to find another fix for that, though.